CONTRIBUTING.rst

```diff
@@ -487,3 +487,13 @@ Before we can accept your pull requests you'll need to sign a Contributor Licens
 - **If you work for a company that wants to allow you to contribute your work**, then you'll need to sign a `corporate CLA <https://developers.google.com/open-source/cla/corporate>`__.
 
 You can sign these electronically (just scroll to the bottom). After that, we'll be able to accept your pull requests.
+
+*******************************
+Helpful Pull Request Guidelines
+*******************************
```

Comment:
Shouldn't this just be the first bullet in the `Contributing` section at the top?